ontobio/ontol.py

```diff
@@ -358,6 +358,73 @@ def node_type(self, id):
         """
         return self.node(id)['type']
 
+    def get_ancestors(self, go_term):
+        """
+        Returns the ancestors from the is_a, part_of relation traversed GO subontology of go_term's ancestors.
+
+        subontology() primarily used here for speed when specifying relations to traverse.
```

Comment:
Yeah, I figured this out a long time ago so it might've changed (or cache is messing me up), but specifying relations without using `subontology` first will calculate the filtered graph using the whole GO ontology rather than just the smaller subset of "all relations" ancestors of a node.

Like, specifiying relations requires looking at edge properties (in `ontol.get_filtered_graph()`) whereas not just cares that an edge is there. Does this make sense?

Reply:
Thank you @dustine32 
Yes, I see the difference and where the bottleneck is 👍